packages/elements/package.json

```diff
@@ -26,6 +26,16 @@
   "author": "Clerk",
   "sideEffects": false,
   "exports": {
+    "./*": {
```

Comment:
All React-specific code is contained within `dist/react` but available at the top-level by default, per previous conversations.